Content.Shared/Hands/HandEvents.cs

```diff
@@ -73,15 +70,9 @@ public sealed class HandDeselectedEvent : HandledEntityEventArgs
         /// </summary>
         public EntityUid User { get; }
 
-        /// <summary>
-        ///     Item in the hand that was deselected.
-        /// </summary>
-        public EntityUid Item { get; }
-
-        public HandDeselectedEvent(EntityUid user, EntityUid item)
```

Comment:
were these just unused?

Reply:
The event was and is currently just raised directed at the item, and is not broadcast. So its currently just completely redundant information.

If ever it gets broadcasted, or this event is split into a "Got-Selected" and "Did-selection" (ie directed at item and at user) then the field would be useful again. But for now just removing it is probably fine?